### ISSUE TITLE
Language detection in the demo is stuck on "Delphi"

### ISSUE THREAD

**Issue by isagalaev** [highlightjs/highlightjs.org#7]
Typing a single character into the code box on https://highlightjs.org/demo results in the output window saying "Delphi" and code not highlighted (even valid Delphi).
![image](https://github.com/highlightjs/highlightjs.org/assets/99931/9f693bb2-a996-4801-bda1-4f5ab6f8e231)
I'm using Firefox, didn't try in other browsers.

**Comment by allejo** [comment]
hmm looks like this is only a problem during production builds, running a local development build works just fine 🤔

**Comment by isagalaev** [comment]
By the way, just checked it happens in Chromium as well, so probably not browser specific.

**Comment by allejo** [comment]
@joshgoebel I investigated this issue more, and it looks like auto-detection is crapping the bed. I haven't tried replicating this in a JSFiddle yet, but do you know what may be causing this? I can't image the Delphi grammar is this fucked with relevancy calculations.

<img width="379" alt="image" src="https://github.com/highlightjs/highlightjs.org/assets/1246453/c015a2c3-2de3-4dbe-81c5-679951794d44">

**Comment by joshgoebel** [comment]
You said it worked in dev mode, are these relevance numbers actually different in dev mode?

I'd remove `(autodetect)` from the dropdown... It was never intended to work well with all 190 grammars loaded... it's probably "better than nothing" to "ok" with just `:common` loaded... but it's not at all a good way to decide what a random snippet of code is.  So there is no fix here that is going to always "just work".

**Comment by allejo** [comment]
Yes, these relevancy numbers are entirely different in development mode (actually sane compared to dev). My current theory is that Webpack is fucking something up in the bundling process, leading to these inconsistencies.

But I agree, I am leaning towards removing the auto-detection for all 190 languages and just using `:common` instead.

**Comment by joshgoebel** [comment]
Sounds like something in the build is indeed broken.  Could you just use a static build instead, like from cdn-release?  Or use a much better bundler, like rollup?
